src/metabase/search/postgres/core.clj

```diff
@@ -91,11 +92,12 @@
 (defn- minimal
   "Search via index, and return potentially stale information, without applying filters or
   restricting to collections we have access to."
```

Comment:
docs need to be updated :D, it does filtering now

Reply:
hehe thanks

Reply:
also, why do you only enable filter for minimal searches, what's about the others?

Reply:
The other ones already having filtering implicitly, because they join to the legacy queries.